string msg_id     # "ACWL0001", "WLAC0001" ...
string date       # YYYYMMDD
string time       # HHMMSS
string form       # 전문 구분
uint16 msg_len    # 전문 전체 길이(숫자화)
string filler     #temp

